package.json

```diff
@@ -8,6 +8,7 @@
   "private": true,
   "dependencies": {
     "@babel/runtime": "7.15.4",
+    "@mattermost/react-native-paste-input": "0.1.5",
```

Comment:
Would you be open to updating NOTICE.txt https://github.com/mattermost/mattermost-mobile/blob/master/NOTICE.txt too? 🙏

Reply:
Is there a need? Package is ours

Reply:
Ok, not 100% sure.

Reply:
Ok, not 100% sure. @jamiehurewitz would you know?

Reply:
@amyblais going to merge, if it ends up being needed, we can add it in another PR

Reply:
I am not fully sure I understand the question, but if there are changes to the components and we need to provide new notices, then we would need to update that everywhere there are changes.

Reply:
@jamiehurewitz let me explain, normally when we add a dependency to the project of a third party library, we add it to the notices file.

In this case, there was a new dependency added, but the dependency is not a third-party own, but own by Mattermost, so the question is if the dependency is own by us, do we still have to add it in the Notices file ?

Reply:
Ah, I see. Thank you, that makes sense. Is this the only place where dependencies are listed? Our users often times ask what dependencies are incorporated into our product and we need a place to point them to. If this is the document we use to point them to, then I would say, yes, add it, even if it is our own, so we can keep an accurate accounting of all dependencies.

Reply:
The actual file that list all dependencies is not the notices file but the package.json and if they need dependencies of dependencies they should look into the package-lock.json